blog/modular-models-announcement.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Modular Models
+description: Modular Models 
+slug: modular-models-announcement
+date: 2024-04-21
+authors: eharris
+tags: [openfga,features]
+image: https://openfga.dev/img/og-rich-embed.png
+hide_table_of_contents: false
+---
+# Release Candidate for Modular Models
+
+Modular models aims to improve the model authoring experience when multiple teams are maintaining a model, such as:
+
+- A model can grow large and difficult to understand
+- As more teams begin to contribute to a model, the ownership boundaries may not be clear and code review processes might not scale
+
+With modular models, a single model can be separated across multiple files allow grouping of types and conditions into modules. This means a model can more easily be organized in terms of team or organizational structure and by using a feature such as [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) or [GitLab](https://docs.gitlab.com/ee/user/project/codeowners/) code owners, it should become easier to ensure the owners of an portion of your model are correctly assigned to review it.
```

Comment:
Consider splitting this up a bit.

```suggestion
With modular models, a single model can be separated across multiple files allow grouping of types and conditions into modules. This means that a model can be organized more easily in terms of team or organizational structure. Used in conjunction with features such as [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) or [GitLab](https://docs.gitlab.com/ee/user/project/codeowners/) code owners, it should become easier to ensure the owners of an portion of your model are correctly assigned to review it.
```

Reply:
Also here we mention GitLab's and GitHub's code owners feature, but in the `modular-models.mdx` file below we _also_ mention Gitea's implementation - should we be consistent here?

Reply:
Yeah will add Gitea here too

```suggestion
With modular models, a single model can be separated across multiple files allow grouping of types and conditions into modules. This means that a model can be organized more easily in terms of team or organizational structure. Used in conjunction with features such as [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners), [GitLab](https://docs.gitlab.com/ee/user/project/codeowners/) or [Gitea's](https://docs.gitea.com/usage/code-owners) code owners. code owners, it should become easier to ensure the owners of an portion of your model are correctly assigned to review it.
```